framework/src/utils/lib/utils.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+import * as fs from 'fs';
+import * as yaml from 'js-yaml';
+
+/**
+ * Utilities class used across the different resources
+ */
+export class Utils {
```

Comment:
It's a bit too generic to me, it will become a big file where we put everything that we don't know where to put...

Are we sure they are used in multiple places (modules)? Do we really need to have them here...